### PR TITLE
U ichanges

### PR DESCRIPTION
**Pull request by rahul-tarafdar-synerzip**

### CODE REVIEW

File: src/browser/modules/EditorInfo/EditNodes.jsx [hunk 1]
```diff
@@ -1,43 +1,24 @@
...
-          {key}:{getStringValue(value)}
-        </div>
-      )
-    })
-
     return (
       <Drawer id='db-drawer'>
         <DrawerHeader>Editor</DrawerHeader>
         <DrawerBody>
           <DrawerSection>
```
Comment (rahul-tarafdar-synerzip) [inline src/browser/modules/EditorInfo/EditNodes.jsx]: remove this section

File: src/browser/modules/EditorInfo/ViewProperties.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  DrawerSection,
+  DrawerSectionBody,
+  DrawerSubHeader
+} from 'browser-components/drawer'
+
+import * as _ from 'lodash'
+import { getStringValue } from './utils'
+import { StyledTable, StyledKey, StyledValue } from '../DatabaseInfo/styled'
+
+export class ViewProperties extends Component {
```
Comment (rahul-tarafdar-synerzip) [inline src/browser/modules/EditorInfo/ViewProperties.jsx]: Why not functional component?

File: src/browser/modules/EditorInfo/EditNodes.jsx (hunk 1 above)
Comment (Gayatriuparate) [reply src/browser/modules/EditorInfo/EditNodes.jsx]: done

File: src/browser/modules/EditorInfo/ViewProperties.jsx (hunk 2 above)
Comment (Gayatriuparate) [reply src/browser/modules/EditorInfo/ViewProperties.jsx]: done

**Review by KirthiNair — APPROVED** [review]

**Pull request merged**